### ISSUE TITLE
test_ngw_7 failure

### ISSUE THREAD

**Issue by rouault** [OSGeo/gdal#2501]
The following test has just started failing:
```
_________________________________ test_ngw_7 __________________________________
    def test_ngw_7():
    
        if gdaltest.ngw_drv is None or gdaltest.ngw_ds is None:
            pytest.skip()
    
        if check_availability(gdaltest.ngw_test_server) == False:
            gdaltest.ngw_ds = None
            gdaltest.ngw_drv = None
            pytest.skip()
    
        gdal.SetConfigOption('CPL_ACCUM_ERROR_MSG', 'ON')
        gdal.PushErrorHandler('CPLQuietErrorHandler')
    
        ovr_band = gdaltest.ngw_ds.GetRasterBand(1).GetOverview(21)
        cs = ovr_band.Checksum()
    
        gdal.PopErrorHandler()
        gdal.SetConfigOption('CPL_ACCUM_ERROR_MSG', 'OFF')
        msg = gdal.GetLastErrorMsg()
        gdal.ErrorReset()
    
        if msg is not None and msg != '':
            print('Last error message: {}.'.format(msg))
            pytest.skip()
    
>       assert cs == 5, 'Wrong checksum: ' + str(cs)
E       AssertionError: Wrong checksum: 2
E       assert 2 == 5
E         -2
E         +5
ngw.py:256: AssertionError
```

**Comment by rouault** [comment]
I've temporarily disabled it in 46ddfaa

**Comment by BishopGIS** [comment]
Ok, let me see. We just updated sandbox.nextgis.com to latest version of NextGIS Web. This may be the root of this problem.

**Comment by rouault** [comment]
@BishopGIS There was a pyflakes issue I've fixed in dcfc36d

**Comment by BishopGIS** [comment]
Thanks!
